internal/apiserver/routes.go

```diff
@@ -30,18 +32,30 @@ var routes = []*oapispec.Route{
 	getEventByID,
 	getEvents,
 	getMsgByID,
+	getMsgData,
 	getMsgEvents,
 	getMsgOps,
+	getMsgTxn,
 	getMsgs,
+	getNetworkOrg,
+	getNetworkOrgs,
+	getNetworkNode,
+	getNetworkNodes,
 	getNamespace,
 	getNamespaces,
 	getOpByID,
 	getOps,
-	getSubsriptions,
 	getSubscriptionByID,
+	getSubsriptions,
```

Comment:
`Subscriptions`

Reply:
👍  - added fix in next PR in the chain: https://github.com/kaleido-io/firefly/pull/42/commits/a8b3e3d12366b99f09cca31b0c3a040c3aab5007